restapi/api_client.go

```diff
@@ -218,7 +219,11 @@ func (client *api_client) send_request(method string, path string, data string)
 		if resp.StatusCode == 301 || resp.StatusCode == 302 {
 			//Redirecting... decrement num_redirects and proceed to the next loop
 			//uri = URI.parse(rsp['Location'])
-		} else if resp.StatusCode == 404 || resp.StatusCode < 200 || resp.StatusCode >= 303 {
+		) else if resp.StatusCode >= 500 {
```

Comment:
I think a check for `>= 500 && < 600` would be most proper since the 600+ range isn't officially in use in the HTTP spec so some implementations have started using it for other stuff.